README.md

```diff
@@ -120,17 +120,23 @@ name: "example rule"
 # exist, the rule applies to every pull request.
 if:
   # "changed_files" is satisfied if any file in the pull request matches any
-  # regular expression in the list.
+  # regular expression in the list. The "ignore" list is optional and will
+  # ignore any files that match the regular expressions listed.
```

Comment:
Maybe `in the "paths" list. If the "ignore" list is present, files in the pull request matching these regular expressions are ignored by this rule.`?

I think clarifying the `paths` list important, but I don't feel as strongly about the rest (I do think it's a bit more explicit about the behavior.)